packages/plugins/Avatar/src/Application/NFTListDialog.tsx

```diff
@@ -196,16 +196,18 @@ export const NFTListDialog: FC = () => {
                         padding: '8px 16px',
                         justifyContent: 'space-between',
                     }}>
-                    {selectedPluginId === NetworkPluginID.PLUGIN_EVM ? (
-                        <Button
-                            variant="text"
-                            size="small"
-                            className={classes.addButton}
-                            disableRipple
-                            onClick={openAddDialog}>
-                            {t.add_collectible()}
-                        </Button>
-                    ) : null}
+                    <div>
+                        {selectedPluginId === NetworkPluginID.PLUGIN_EVM ? (
+                            <Button
+                                variant="text"
+                                size="small"
+                                className={classes.addButton}
+                                disableRipple
+                                onClick={openAddDialog}>
+                                {t.add_collectible()}
+                            </Button>
+                        ) : null}
+                    </div>
```

Comment:
Put the `<div />` inside the condition. Otherwise, you will have an empty `<div />` when the condition fails.

```suggestion

                        {selectedPluginId === NetworkPluginID.PLUGIN_EVM ? (
                                            <div>
                            <Button
                                variant="text"
                                size="small"
                                className={classes.addButton}
                                disableRipple
                                onClick={openAddDialog}>
                                {t.add_collectible()}
                            </Button>
                                                </div>
                        ) : null}
```

Reply:
div is placeholder

Reply:
Then use:

```
condition ? <Button /> : <div />
```